rootfs/api/models/app.py

```diff
@@ -775,3 +785,27 @@ def _scheduler_filter(self, **kwargs):
             labels.update({'type': kwargs['type']})
 
         return labels
+
+    def _build_env_vars(self, build_type, version, envs):
+        """
+        Build a dict of env vars, setting default vars based on app type
+        and then combining with the user set ones
+        """
+
+        # mix in default environment information deis may require
+        default_env = {
+            'DEIS_APP': self.id,
+            'WORKFLOW_RELEASE': version
+        }
+
+        # Check if it is a slug builder image.
+        if build_type == 'buildpack':
+            # overwrite image so slugrunner image is used in the container
+            default_env['SLUG_URL'] = settings.SLUGRUNNER_IMAGE
```

Comment:
this is not slugrunner_image....this should be the path of the app slug to be run